applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -860,6 +860,9 @@ play_messages([H|T]=Messages, PrevMessages, Count, Box, Call) ->
         {'ok', 'delete'} ->
             lager:info("caller chose to delete the message"),
             _ = kapps_call_command:b_prompt(<<"vm-deleted">>, Call),
+            MessageId = kz_json:get_ne_binary_value(<<"media_id">>, H),
+            JObj = hd(kz_json:get_list_value(<<"succeeded">>, kvm_messages:fetch(AccountId, [MessageId], BoxId))),
+            'ok' = kvm_util:publish_voicemail_deleted(BoxId, JObj, 'dtmf'),
```

Comment:
No need to pattern match `'ok'` here. Dialyzer won't complain about unused return value if it is a simple atom (like ok or boolean).

Reply:
We will update for this.